driver/random.hpp

```diff
@@ -1,16 +1,28 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
-#include <cstdlib>
+#include <random>
+
+std::minstd_rand& get_minstd_gen()
+{
+    static std::random_device rd;
+    static std::minstd_rand minstd_gen(rd());
+
+    return minstd_gen;
+}
 
 template <typename T>
 inline T FRAND()
 {
-    double d = static_cast<double>(rand() / (static_cast<double>(RAND_MAX)));
+    double d = std::generate_canonical<double, 1>(get_minstd_gen());
     return static_cast<T>(d);
 }
 
-inline int GET_RAND() { return rand(); }
+inline int GET_RAND()
+{
+    auto minstd_gen = get_minstd_gen();
```

Comment:
`auto &` since default `auto` does not inherit reference.

Reply:
done!